File: integers.cpp
#include <iostream>
using namespace std;

int main() {
    int numbers[20];
    int even[20], odd[20];
    int evenCount = 0, oddCount = 0;

for (int i = 0; i < 20; i++) {
    numbers[i] = (i + 1);  
}
for (int i = 0; i < 20; i++) {
if (numbers[i] % 2 == 0) {
even[evenCount] = numbers[i];
    evenCount++;}
else {
odd[oddCount] = numbers[i];
    oddCount++;
}
}
cout << "Even numbers: ";
for (int i = 0; i < evenCount; i++) {
cout << even[i] << " ";
}
cout << "\nOdd numbers: ";
for (int i = 0; i < oddCount; i++) {
cout << odd[i] << " ";
}
return 0;
}

File: integers.py
numbers = []
even = []
odd = []

for i in range(20):
    numbers.append(i + 1)
for num in numbers:
 if num % 2 == 0:
   even.append(num)
 else:
   odd.append(num)
print("Even numbers:", even)
print("Odd numbers:", odd)
